### PR TITLE
Sync kubeflow model registry manifests v0.2.1-alpha

### PR DESCRIPTION
**Pull request by tarilabs**
# Pull Request Template for Kubeflow manifests Issues

- fix `sync-model-registry-manifests.sh` errors
- sync Manifests
- add to test following up on https://github.com/kubeflow/model-registry/pull/101

## ✏️ A brief description of the changes
per description above

## 📦 List any dependencies that are required for this change
none

## 🐛 If this PR is related to an issue, please put the link of the issue here.
none

## ✅ Unit Test Checklist

these manifests are tested in KF/model-registry repo.
the test addition was demonstrated with: https://github.com/kubeflow/model-registry/pull/101

## ✅ Contributor checklist
  - [x] All the commits have been _signed-off_  (To pass the `DCO` check)
  - [ ] Submit the [Contributor License Agreements](https://cla.developers.google.com/clas) (To pass the `cla/google` check)

cc @lampajr

### CODE REVIEW

**Comment by lampajr** [comment]
Thanks @tarilabs !

/lgtm

**Comment by lampajr** [comment]
Tested locally as well and it looks good!

/lgtm

File: .github/workflows/model_registry_test.yaml [hunk 1]
```diff
@@ -42,3 +51,25 @@ jobs:
...
+    - name: Port forward Istio gateway
+      run: |
+        INGRESS_GATEWAY_SERVICE=$(kubectl get svc --namespace istio-system --selector="app=istio-ingressgateway" --output jsonpath='{.items[0].metadata.name}')
+        nohup kubectl port-forward --namespace istio-system svc/${INGRESS_GATEWAY_SERVICE} 8080:80 &
+        while ! curl localhost:8080; do echo waiting for port-forwarding; sleep 1; done; echo port-forwarding ready
+
+    - name: Dry-run KF Model Registry REST API
+      run: |
+        echo "Dry-run KF Model Registry REST API..."
+        export KF_TOKEN="$(kubectl -n default create token default)"
```
Comment (juliusvonkohout) [inline .github/workflows/model_registry_test.yaml:74]: We should probably user a proper kubeflow user namespace instead of just "default"

Comment (tarilabs) [reply .github/workflows/model_registry_test.yaml:74]: this is exactly per KServe test:

https://github.com/kubeflow/manifests/blob/0805068fbcc5f0cde54f1a5009d8ddbba6a8e1c3/.github/workflows/kserve_m2m_test.yaml#L73

Comment (juliusvonkohout) [reply .github/workflows/model_registry_test.yaml:74]: Probably something we should fix there as well. But that is for another PR @lampajr Ill approve for now.

**Comment by tarilabs** [comment]
thanks @juliusvonkohout @lampajr !! 🚀 👍

**Pull request merged**